pages/_app.tsx

```diff
@@ -1,13 +1,19 @@
 import 'tailwindcss/tailwind.css';
 
 import type { AppProps } from 'next/app';
+import Head from 'next/head';
 import { CookiesProvider } from 'react-cookie';
 
 function MyApp({ Component, pageProps }: AppProps) {
     return (
-        <CookiesProvider>
-            <Component {...pageProps} />
-        </CookiesProvider>
+        <>
+            <Head>
+                <title>Nak Naksin!</title>
+            </Head>
+            <CookiesProvider>
+                <Component {...pageProps} />
+            </CookiesProvider>
+        </>
```

Comment:
Think we can put <Head> inside <CookiesProvider> instead
Perhaps in the future, we would like to access cookies within it

Reply:
you mean put the Head tag in the Cookies Providers tag?

Reply:
ya

Reply:
resolve in 
https://github.com/nubpro/nakvaksin/pull/14/commits/48b232c3aee1eaa3e7898161bb84293623706916